modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/suggestions/spi/asah/individuals/AsahRecentlyViewedIndividualsContributor.java

```diff
@@ -71,6 +91,9 @@ protected String getText(JSONObject itemJSONObject) {
 		return itemJSONObject.getString("assetTitle");
```

Comment:
Is getText() used somewhere?

Reply:
modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/suggestions/spi/asah/BaseAsahSuggestionsContributor.java